metadata_backend/conf/conf.py

```diff
@@ -183,11 +183,13 @@ def create_db_client() -> AsyncIOMotorClient:
     "catalog_pid": "urn:nbn:fi:att:data-catalog-sd",
 }
 
-METAX_REFERENCE_FILE = Path(__file__).parent.parent / "conf" / "metax_references" / "metax_references.json"
-if METAX_ENABLED and not METAX_REFERENCE_FILE.is_file():
-    raise RuntimeError(
-        "You must generate the metax references to run submitter: `bash scripts/metax_mappings/fetch_refs.sh`"
-    )
+file_names = ["identifier_types.json", "languages.json", "fields_of_science.json"]
+METAX_REFERENCE_FILES = [Path(__file__).parent.parent / "conf" / "metax_references" / file for file in file_names]
```

Comment:
While this is nice, I think we could have a more verbose version of this command here, for readability's sake. For e.g., with loading the files here. Should probably keep the error checking. Please, notice I wrote incomplete code.
```python
METAX_REFERENCE_ROOT = Path(__file__).parent.parent / "conf" / "metax_references"
# ...
for ref_file in file_names:
    with open(METAX_REFERENCE_ROOT / ref_file) as file:
        METAX_REFERENCE[ref_file.replace(".json", "")] = file.read()
```

Reply:
Thanks for the example